docs/core/config/auth/databases/ldap.md

```diff
@@ -657,14 +651,8 @@ Example: `iterate_filter = (objectClass=smiMessageRecipient)`
 
 LDAP protocol version to use. Likely `2` or `3`.
 
-### `ldaprc_path`
-
-- Default: `<empty>`
-- Values: [[link,settings_types_string]]
-
-If a non-empty value is set, it will be set to the LDAPRC environment variable.
-
-### `pass_attrs`
+### `fields`
+FIXME: THIS IS NOW A BLOCK.
```

Comment:
FIXME. Also, this isn't ldap-specific. And it's really called passdb_fields & userdb_fields